test/plugin/test_output.rb

```diff
@@ -820,6 +823,7 @@ def waiting(seconds)
       assert try_write_called
 
       i.stop; i.before_shutdown; i.shutdown; i.after_shutdown; i.close; i.terminate
+      FileUtils.rm_rf(tmp_dir)
```

Comment:
It should be done in `ensure` block or `teardown` to make sure to delete it even if assertions fails.